inttest/common/footloosesuite.go

```diff
@@ -204,17 +196,8 @@ func (s *FootlooseSuite) SetupSuite() {
 		// Replace the done context with a fresh one.
 		ctx, cancel := newSuiteContext(t)
```

Comment:
We could probably simply use 

```suggestion
		ctx, cancel := context.WithCancel(context.Background())
```

here.

The main difference is that if the test really times out because of the `-timeout` parameter, we'd see a panic instead of "nice" error messages. The `newSuiteContext` is reserving some extra time for the cleanup, and will timeout quite some time before the Go test framework will terminate the test. That might be too early for the cluster state collection.